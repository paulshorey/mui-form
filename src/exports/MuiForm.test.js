import $ from 'jquery'; 
import 'window/theme';

import React from 'react';
import renderer from 'react-test-renderer';

import { MuiForm } from './MuiForm';

it('renders correctly', () => {

    class Result extends React.Component {
        constructor(){
            super();
            this.state = {};
        }
		render() {
			return (
				<MuiForm stateScope={this}></MuiForm>
			);
		}
    }
    const tree = renderer.create(<Result />).toJSON();
    
	expect(tree).toMatchSnapshot();
});
