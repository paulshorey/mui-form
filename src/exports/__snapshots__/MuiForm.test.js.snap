// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiForm_customScrollbars sc-bwzfXH kSxcrS"
>
  <form
    autoComplete="off"
    className="MuiForm sc-htpNat fQHmxG"
    onBlur={[Function]}
    onChange={[Function]}
    onSubmit={[Function]}
    style={
      Object {
        "overflowY": "scroll",
      }
    }
  >
    <div
      className="formSubmitSpacer"
    />
    <div
      className="formSubmit"
    >
      <button
        className=" primary sc-bdVaJa dnAqev"
        disabled={true}
        type="submit"
      >
        <b>
          Save Changes
        </b>
      </button>
    </div>
    <button
      className="clear scrollCue sc-bdVaJa dnAqev"
      onClick={[Function]}
      type="button"
    >
      <b>
        <span
          className="fontIcon icon-carat_down"
        />
      </b>
    </button>
  </form>
</div>
`;
